src/dbus_api/pool/pool_2_0/methods.rs

```diff
@@ -285,11 +285,11 @@ pub fn rename_pool(m: &MethodInfo<MTFn<TData>, TData>) -> MethodResult {
         .expect("implicit argument must be in tree");
     let pool_uuid = get_data!(pool_path; default_return; return_message).uuid;
 
-    let msg = match dbus_context
-        .engine
-        .borrow_mut()
-        .rename_pool(pool_uuid, new_name)
-    {
+    let mut engine = dbus_context.engine.borrow_mut();
+
+    let result = pool_op_logging!("preparing to rename pool with name {}", new_name; "successfully renamed pool with name {}, uuid {}", new_name; engine.rename_pool(pool_uuid, new_name));
```

Comment:
I think it makes sense to include the pool UUID in the pre-message, if possible.

Reply:
Done.